turbo.json

```diff
@@ -41,6 +41,10 @@
     "test:rust": {
       "dependsOn": ["compile"],
       "inputs": ["crate/src/**", "crate/Cargo.toml", "crate/Cargo.lock", "crate/tests/**"]
+    },
+    "test:rust-unit": {
+      "dependsOn": ["compile"],
```

Comment:
todo: these tests should not require compile to run. Compile step is only for those tests that require the wasm-pack wasm binaries to be built.

Reply:
Removed the depends on compile.